docs/.doxygen/Doxyfile

```diff
@@ -0,0 +1,59 @@
+# Auto-generated doxygen configuration file
+
+BUILTIN_STL_SUPPORT = YES
+
+CALLER_GRAPH = YES
+
+CALL_GRAPH = YES
+
+ENUM_VALUES_PER_LINE = 1
+
+EXCLUDE_PATTERNS = ../../src/targets/gpu/kernels ../../src/targets/gpu/device
+
+EXTRACT_ALL = YES
+
+FULL_PATH_NAMES = YES
+
+GENERATE_LATEX = NO
+
+GENERATE_TREEVIEW = YES
+
+GENERATE_XML = YES
+
+INCLUDE_PATH = ../../src/include ../../src/targets/cpu/include ../../src/targets/gpu/include
+
+INPUT = ../../src
+
+MACRO_EXPANSION = YES
+
+OUTPUT_DIRECTORY = docBin
```

Comment:
File paths should be all lowercase.

Reply:
For this, rocm-docs-core looks for the files in the docBin folder as an intermediate for its doxygen sub-extension. This folder won't be committed to the repository.